flink-connector/src/main/java/com/datastrato/gravitino/flink/connector/catalog/GravitinoCatalogManager.java

```diff
@@ -64,9 +64,11 @@ public static GravitinoCatalogManager get() {
    * <p>After close, GravitinoCatalogManager can not be used anymore.
    */
   public void close() {
-    Preconditions.checkState(!isClosed, "Gravitino Catalog is already closed");
-    isClosed = true;
-    gravitinoClient.close();
+    if (!isClosed) {
```

Comment:
why changing this?

Reply:
Why `gravitinoCatalogManager` may be closed and recreated again?

Reply:
@FANNG1 The gravitinoCatalogManager is managed by CatalogStore and CatalogStore will always be closed when invoking the `stop()` in FlinkEnvIT. When multiple tests inherit the FlinnkEnvIT, the gravitinoCatalogManager will be closed and recreated.

Reply:
got it